client/src/app/gateways/notify.service.ts

```diff
@@ -267,6 +264,7 @@ export class NotifyService {
         if (channels) {
             notify.to_channels = channels;
         }
+        console.debug(`Send following data over ICC:`, PUBLISH_PATH, notify);
```

Comment:
should we really merge this line?

Reply:
I'll merge this PR now to test it in new update in production mode. - @jsangmeister Please answer after merge.